ddtrace/llmobs/_integrations/langchain.py

```diff
@@ -20,6 +28,114 @@
 class LangChainIntegration(BaseLLMIntegration):
     _integration_name = "langchain"
 
+    def llmobs_set_tags(
+        self,
+        operation: str,  # oneof "llm","chat_model","chain"
+        span: Span,
+        inputs: Any,
+        response: Any,
+        error: bool = False,
+    ) -> None:
+        """Sets meta tags and metrics for span events to be sent to LLMObs."""
+        if not self.llmobs_enabled:
+            return
+        model_provider = span.get_tag(PROVIDER)
+        span.set_tag_str(SPAN_KIND, "llm")
+        span.set_tag_str(MODEL_NAME, span.get_tag(MODEL) or "")
+        span.set_tag_str(MODEL_PROVIDER, model_provider or "")
+
+        # input parameters
+        if model_provider:
+            input_parameters = {}
+            temperature = span.get_tag(f"langchain.request.{model_provider}.parameters.temperature")
+            max_tokens = span.get_tag(f"langchain.request.{model_provider}.parameters.max_tokens")
+            if temperature:
+                input_parameters["temperature"] = float(temperature)
+            if max_tokens:
+                input_parameters["max_tokens"] = int(max_tokens)
+            if input_parameters:
+                span.set_tag_str(INPUT_PARAMETERS, json.dumps(input_parameters))
+
+        if operation == "llm":
+            self._llmobs_set_meta_tags_from_llm(span, inputs, response, error)
+        elif operation == "chat_model":
+            self._llmobs_set_meta_tags_from_chat_model(span, inputs, response, error)
+        elif operation == "chain":
+            # TODO to be added as a follow-up PR
```

Comment:
For cleanliness sake, let's avoid leaving TODOs in code, and keep track of this task on our JIRA instead